php/README.md

```diff
@@ -4,6 +4,22 @@ This directory contains example Ice applications written in PHP. Each example is
 that demonstrates a specific feature or programming technique. The corresponding server is provided by the C++, C#,
 Java, and Python demos.
 
+> Ice for PHP is only supported on Linux and macOS platforms.
+
+## Linux Prerequisites
+
+- A supported Linux Distribution (RHEL 9/10, Amazon Linux 2023, Debian 12, Ubuntu 24.04)
+- [Nightly build of Ice](https://github.com/zeroc-ice/ice/blob/main/NIGHTLY.md#linux)
+
+## macOS Prerequisites
+
+- macOS Sequoia
+- Ice for PHP source build, see [building Ice for PHP](https://github.com/zeroc-ice/ice/blob/main/php/BUILDING.md)
```

Comment:
No nightly build for macOS?

Reply:
The brew tap doesn't include Ice for PHP.